core/controller/src/main/scala/whisk/core/controller/WebActions.scala

```diff
@@ -700,9 +713,11 @@ trait WhiskWebActionsApi extends Directives with ValidateRequestSize with PostAc
    */
   private def actionLookup(actionName: FullyQualifiedEntityName)(
     implicit transid: TransactionId): Future[WhiskActionMetaData] = {
-    getAction(actionName) recoverWith {
-      case _: ArtifactStoreException | DeserializationException(_, _, _) =>
-        Future.failed(RejectRequest(NotFound))
+    resolveAction(actionName) flatMap { resolveAction =>
+      getAction(resolveAction) recoverWith {
```

Comment:
i dont believe this is right - the resolveAction method returns just the name of the action in the base package, and so getting that action means you'll miss the bound parameters. I think you want to use resolveActionAndMergeParameters instead.

Reply:
Oh... I understand I'll modify it soon.